pkg/catalog/types.go

```diff
@@ -0,0 +1,28 @@
+package catalog
+
+import (
+	imagev1 "github.com/openshift/api/image/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Catalog ...
+type Catalog struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	Spec              CatalogSpec `json:"spec,omitempty"`
+}
+
+// CatalogSpec ...
+type CatalogSpec struct {
+	Namespace      string              `json:"namespace"`
```

Comment:
namespace field is already in `metav1.ObjectMeta` (`metadata`) we should use that one instead of creating a new duplicate field.

Reply:
Heck. Completely forgot about this part.